### PR TITLE
Added Reyts Registration Details

### PR DESCRIPTION
**Pull request by reyts**
Added Reyts Registration details

### CODE REVIEW

**Comment by reyts** [comment]
Added Reyts Registration details

File: registry.json [hunk 1]
```diff
@@ -3871,5 +3871,19 @@
...
+    "ticker": "REYTS",
+    "poolId": "b3471fb159f5e8b345018a5c309afd91f06bf99552b8e91eb9980c9d",
+    "social": {
+      "website": "https://www.reyts.com/reytspool/",
+      "twitter": "Reytsapp",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "",
```
Comment (Bez625) [inline registry.json:3885]: You must have a telegram handle to join the alliance. This must be the telegram of an individual and not a group or channel

File: registry.json [hunk 2]
```diff
@@ -3871,5 +3871,19 @@
...
+    "operatorName": "Reyts",
```
Comment (Bez625) [inline registry.json:3886]: Please change this to be either your real full name, real first name or leave blank

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by Bez625** [comment]
If you would still like to be included please can you make the changes requested? I will close in the coming weeks if no changes are made

**Comment by Bez625** [comment]
Closing as no update for a while. If you would still like to be added to the alliance then please feel free to raise another PR

**Pull request closed without merging**